libs/axiosBuildScripts.js

```diff
@@ -11,14 +11,52 @@ const versionsRef = require('../docs/.vitepress/versions.json');
  * Builds the list of chains for the dAPIs docset
  */
 async function dapiChains() {
-  const response = await axios.get(
-    'https://db-api-prod.api3.org/api/docs-chains-reference'
+  /*
+    DEPRECATED
+    const response = await axios.get(
+      'https://db-api-prod.api3.org/api/docs-chains-reference'
+    );
+    const chains = response.data;
+  */
+
+  const repo = await axios.get(
+    'https://raw.githubusercontent.com/api3dao/airnode-protocol-v1/main/deployments/references.json'
   );
-  const chains = response.data;
+  const repoData = repo.data;
+
+  let list = {};
+  Object.entries(repoData.chainNames).forEach((element) => {
+    // From the repo
+    const alias = element[1];
+    const id = element[0];
+    const contractList = {
+      Api3ServerV1: repoData.Api3ServerV1[id],
+      AccessControlRegistry: repoData.AccessControlRegistry[id],
+      OwnableCallForwarder: repoData.OwnableCallForwarder[id],
+      ProxyFactory: repoData.ProxyFactory[id],
+    };
+
+    // From @api3/chains
+    const name = CHAINS.find((x) => x.id === id).name;
+    const nativeToken = CHAINS.find((x) => x.id === id).symbol;
+    const testnet = CHAINS.find((x) => x.id === id).testnet;
+    const explorerUrl = CHAINS.find((x) => x.id === id).explorer.browserUrl;
```

Comment:
I think this could be simpler by searching for the chain once and storing the result in a variable:
```
    // From @api3/chains
    const chain = CHAINS.find((x) => x.id === id);
```

Then, since each of these subsequently defined variables are only used once, update the `list[alias]` below for each e.g. `name: chain.name`

Reply:
Guess I was asleep at the wheel on this one. Thanks for the catch.